src/components/sbb-image/sbb-image.stories.js

```diff
@@ -4,14 +4,16 @@ import readme from './readme.md';
 
```

Comment:
At the moment there is only one story, maybe it would be nice generate some more stories so that they are covered by chromatic visual tests. (e.g. radius property set to false, or one without caption text)

Reply:
when needed, we will do.

Reply:
It helps for visual regression tests which would prevent unwanted changes in future, but it's not a hard rule. On the other hand, the workload is not that hard.

Reply:
I added.